lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -1219,7 +1152,7 @@ void ReabstractionInfo::specializeConcreteAndGenericSubstitutions(
         M,
         [&](SubstitutableType *type) -> Type {
           auto SpecializedInterfaceTy =
-              CalleeInterfaceToSpecializedInterfaceType(type);
+              MapCalleeInterfaceTypeToSpecializedInterfaceType(type);
 
           return SpecializedGenericEnv->mapTypeOutOfContext(
```

Comment:
I'm not sure about this. We need to gat an interface type at the end, right? If we use SpecializedInterfaceToCallerArchetypeMap, we are going to get a contextual type, or?